components/log-viewer-webui/client/src/main.tsx

```diff
@@ -1,19 +1,33 @@
 import {StrictMode} from "react";
 import {createRoot} from "react-dom/client";
 
+import AntdApp from "./AntdApp";
 import App from "./App";
 
 import "./index.css";
 
 
+/* eslint-disable-next-line no-warning-comments */
+// TODO: Remove flag and related logic when the new UI is fully implemented.
+const antdFlag = false;
+
 const rootElement = document.getElementById("root");
 if (null === rootElement) {
     throw new Error("Root element not found");
 }
 
 const root = createRoot(rootElement);
-root.render(
-    <StrictMode>
-        <App/>
-    </StrictMode>
-);
+/* eslint-disable-next-line @typescript-eslint/no-unnecessary-condition */
+if (antdFlag) {
+    root.render(
+        <StrictMode>
+            <AntdApp/>
```

Comment:
To avoid duplicate code, can we write
```
const AppComponent = (condition)? AntdApp: App;
...
root.render(
    <StrictMode>
        <AppComponent/>
    </StrictMode>
)
```

I imagine we might be adding other wrapping components later, but until then let's keep it concise.